src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -1542,36 +1621,81 @@ public C(string s)
     {{
         if (string.IsNullOrWhiteSpace(s))
         {{
-            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_whitespace, "{nameof(s)}")}"", nameof(s));
+            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_whitespace, "{nameof(s)}").Replace("\"", "\\\"")}"", nameof(s));
+        }}
+    }}
+}}",
+                CodeActionIndex = 2
+            }.RunAsync();
+        }
+
+        [WorkItem(51338, "https://github.com/dotnet/roslyn/issues/51338")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
+        public async Task TestSpecialStringCheck3()
+        {
+            var culture = CultureInfo.CurrentUICulture;
```

Comment:
❔ Can we not use `UseCultureAttribute`?

Reply:
I don't know, I didn't know it existed :)